### PR TITLE
refactor: feature page components, double border issues

### PR DESCRIPTION
**Pull request by manika-signoz**
## Pull Request

---

#### Screenshots / Screen Recordings (if applicable)

| Page | Before | After |
|------|--------|-------|
| Agent Native Observability | agent-native-observability-before | agent-native-observability-after |
| Alerts Management | alerts-management-before | alerts-management-after |
| ClickStack Alternative | clickstack-alternative-before | clickstack-alternative-after |
| Cloudwatch Alternative | cloudwatch-alternative-before | cloudwatch-alternative-after |
| Datadog Migration Tool | datadog-migration-tool-before | datadog-migration-tool-after |
| Distributed Tracing | distributed-tracing-before | distributed-tracing-after |
| External APIs | external-apis-before | external-apis-after |
| LLM Observability | llm-observability-before | llm-observability-after 
| Log Management |
log-management-before | log-management-after 
| Pricing | pricing-before | pricing-after |
| Trace Funnels | trace-funnels-before | trace-funnels-after |

---

### ✅ Change Type
_Select all that apply_

- [ ] ✨ Feature
- [ ] 🐛 Bug fix
- [x] ♻️ Refactor
- [ ] 🛠️ Infra / Tooling
- [ ] 🧪 Test-only

---

- Tests added/updated:
- Manual verification:
- Edge cases covered:

---

- Blast radius: Feature pages
- Potential regressions: UX regressions on Feature pages
- Rollback plan: Revert the PR

---

### 📋 Checklist
- [x] Tests added or explicitly not required
- [x] Manually tested
- [x] Breaking changes documented
- [x] Backward compatibility considered

---

### CODE REVIEW

File: app/(site)/distributed-tracing/DistributedTracingPage.constants.tsx [hunk 1]
```diff
@@ -10,117 +10,221 @@ import {
...
+import { BUTTON_CLASS_NAME } from '@/shared/components/molecules/FeaturePages/constants'
 
 export const CLOUD_ICONS = [
-  { src: "/img/icons/aws-icon.svg", alt: "AWS" },
-  { src: "/img/icons/gcp-icon.svg", alt: "Google Cloud" },
-  { src: "/img/icons/azure-icon.svg", alt: "Azure" }
-];
+  { src: '/img/icons/aws-icon.svg', alt: 'AWS' },
+  { src: '/img/icons/gcp-icon.svg', alt: 'Google Cloud' },
+  { src: '/img/icons/azure-icon.svg', alt: 'Azure' },
```
Comment (H4ad) [inline app/(site)/distributed-tracing/DistributedTracingPage.constants.tsx]: follow-up: use ?url imports for this and others.

File: shared/components/molecules/FeaturePages/SectionLayout/SectionLayout.view.tsx [hunk 2]
```diff
@@ -9,23 +10,23 @@ const SectionLayout: React.FC<SectionLayoutProps> = ({
...
+        return '!mx-auto'
       case 'bordered':
-        return '!mx-auto border border-dashed !border-b-0 !border-t-0 border-signoz_slate-400 md:!w-[80vw]'
+        return '!mx-auto border border-dashed !border-b-0 !border-t-0 border-signoz_slate-400'
       case 'no-border':
-        return '!mx-auto !w-[90vw] border !border-b-0 !border-t-0 border-none border-signoz_slate-400 md:!w-[80vw]'
+        return '!mx-auto !w-[90vw]'
       case 'border-x':
-        return '!mx-auto !w-[90vw] border border-dashed border-b-1 border-t-1 border-signoz_slate-400 md:!w-[80vw]'
+        return '!mx-auto !w-[90vw] border border-dashed border-b-1 border-t-1 border-signoz_slate-400'
```
[the remark this answers is not in the document]
Comment (manika-signoz) [reply shared/components/molecules/FeaturePages/SectionLayout/SectionLayout.view.tsx]: done

File: shared/components/molecules/FeaturePages/SignozStats/SignozStats.view.tsx [hunk 3]
```diff
@@ -89,7 +89,7 @@ const SigNozStats = () => {
               ))}
             </div>
           </div>
-          <div className="[&>div]:!border-r-1 grid grid-cols-1 text-left sm:grid-cols-2 [&>div]:border-l-0 [&>div]:border-signoz_slate-400">
+          <div className="[&>div]:!border-r-1 grid grid-cols-1 text-left md:grid-cols-2 [&>div]:border-l-0 [&>div]:border-signoz_slate-400">
```
[the remark this answers is not in the document]
Comment (manika-signoz) [reply shared/components/molecules/FeaturePages/SignozStats/SignozStats.view.tsx]: fixed

File: shared/components/molecules/FeaturePages/Divider/Divider.view.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { cn } from 'app/lib/utils'
+import { DividerProps } from './Divider.types'
+
+const Divider: React.FC<DividerProps> = ({
+  orientation = 'horizontal',
+  variant = 'dashed',
+  className,
+}) => {
+  return (
+    <div
```
[the remark this answers is not in the document]
Comment (manika-signoz) [reply shared/components/molecules/FeaturePages/Divider/Divider.view.tsx:10]: done

File: shared/components/molecules/FeaturePages/FeatureCard/FeatureCard.view.tsx [hunk 5]
```diff
@@ -1,19 +1,8 @@
...
-}) => {
-  const borderClass =
-    variant === 'combined'
-      ? 'border-none'
-      : 'border-r max-md:border-l border-signoz_slate-400 border-dashed last:border-r-0'
-
+const FeatureCard: React.FC<FeatureCardProps> = ({ icon, title, description, className = '' }) => {
   return (
-    <div className={`p-0 ${borderClass} bg-transparent ${className}`}>
+    <div className={`bg-transparent p-0 ${className}`}>
```
[the remark this answers is not in the document]
Comment (manika-signoz) [reply shared/components/molecules/FeaturePages/FeatureCard/FeatureCard.view.tsx]: done

File: shared/components/molecules/FeaturePages/SectionLayout/SectionLayout.view.tsx [hunk 6]
```diff
@@ -9,23 +10,23 @@ const SectionLayout: React.FC<SectionLayoutProps> = ({
...
       default:
-        return '!mx-auto border border-dashed !border-b-0 border-signoz_slate-400 md:!w-[80vw]'
+        return '!mx-auto border border-dashed !border-b-0 border-signoz_slate-400'
```
[the remark this answers is not in the document]
Comment (manika-signoz) [reply shared/components/molecules/FeaturePages/SectionLayout/SectionLayout.view.tsx]: added !border-t-0

File: app/(site)/distributed-tracing/DistributedTracingPage.constants.tsx (hunk 1 above)
Comment (manika-signoz) [reply app/(site)/distributed-tracing/DistributedTracingPage.constants.tsx]: done

**Review by H4ad — APPROVED** [review]

**Comment by GeekBoySupreme** [comment]
@manika-signoz these look so much better. I just have one small concern, take it up whenever you have the bandwidth.

The container width of the page content, the header, and the footer should ideally be uniform. As in, if I draw red keylines on the extreme left and right edges vertically, the contents (in the case of header and footer, the text itself, and in the case of body, the left dashed line or content, whichever is applicable based on the section) should all start from the left keyline and end at the right. I noticed some deviation from this in the Agent-native page hero section (the headline and the graphic have bit more left spacing) and the footer. Could you fix these?

happy to hop on a call to clarify this if necessary.

image
